script/main.py

```diff
@@ -190,11 +190,30 @@ def init_db(conn):
     create_users_table(conn)    
   
   insert_users_to_db(conn, people)
-  
 
-def percentage():
-  print('A function summarizing the percentage of women / men in the database')
 
+def select_all_gender(conn):
+  cur = conn.cursor()
+  cur.execute("SELECT gender FROM users")
+  gender_row = cur.fetchall()  
+  return gender_row
+
+
+def percentage(conn):
+  gender_row = select_all_gender(conn)
+  
+  male = 0
+  female = 0
+  for field in gender_row:
+    if field == ('male',):
```

Comment:
It would be better to have comparison like
```
if field == 'male'
```
But what kind of data type if field? And how to fix it?

Reply:
this is a list, I do not know why it is downloaded in this format ... in the database it is normally written "male"

Reply:
If this is a list then you should know hot to compare to 'male' instead of ('male',)